airflow/providers/google/provider.yaml

```
@@ -984,6 +984,8 @@ extra-links:
   - airflow.providers.google.cloud.links.cloud_memorystore.MemcachedInstanceListLink
   - airflow.providers.google.cloud.links.cloud_memorystore.RedisInstanceDetailsLink
   - airflow.providers.google.cloud.links.cloud_memorystore.RedisInstanceListLink
+  - airflow.providers.google.cloud.links.life_sciences.LifesciencesLink
```

```suggestion
  - airflow.providers.google.cloud.links.life_sciences.LifesciencesLink
```
Looks like a duplicate of sorts and link is named `LifeSciencesLink` which has an entry on the next line.